packages/google_maps_flutter/google_maps_flutter_web/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.1.0+5
+
+* Update `package:google_maps` to `^3.4.5`.
+* Fix GoogleMapController.getLatLng(). [Issue](https://github.com/flutter/flutter/issues/67606).
+* Make InfoWindow contents clickable so onTap works as advertised. [Issue](https://github.com/flutter/flutter/issues/67289).
```

Comment:
```suggestion
* Make `InfoWindow` contents clickable so onTap works as advertised. [Issue](https://github.com/flutter/flutter/issues/67289).
```

Reply:
Thanks for sending suggestions! I'll commit all of these alongside the other fixes so I don't trigger CI multiple times!